locations/spiders/mcdonalds_bg.py

```diff
@@ -1,40 +1,46 @@
-import re
+import json
 
-import scrapy
+from scrapy import FormRequest, Spider
 
-from locations.items import Feature
+from locations.categories import apply_yes_no
+from locations.dict_parser import DictParser
 from locations.spiders.mcdonalds import McDonaldsSpider
 
 
-class McDonaldsBGSpider(scrapy.Spider):
+class McDonaldsBGSpider(Spider):
     name = "mcdonalds_bg"
     item_attributes = McDonaldsSpider.item_attributes
     allowed_domains = ["mcdonalds.bg"]
-    start_urls = ("http://mcdonalds.bg/map/",)
+    start_urls = ["http://mcdonalds.bg/restaurants/"]
```

Comment:
Seemingly this could be `https` instead of `http`?

Reply:
Added